chain/jsonrpc/src/lib.rs

```diff
@@ -1489,6 +1502,10 @@ pub fn start_http(
             .service(last_blocks_html)
             .service(web::resource("/debug/api/sync_info").route(web::get().to(sync_info_handler)))
             .service(sync_info_html)
+            .service(
+                web::resource("/debug/api/chain_info").route(web::get().to(chain_info_handler)),
```

Comment:
I think we start to abuse the "separate APIs"s too much :-)

as we end up with 3 apis (debug/api/sync_info, last_blocks and chain_info) - that return exactly the same JSON.

One option it simply to rename it to "debug/api/status" and then have all the page use this.

The other option is for each of the APIs to return a different object - but that might be a lot more work for now.

Reply:
Cool, I think it is the time to do the refactor as well. Let me proceed with that refactor.